site/en/reference/cli/concepts/udp.md

```diff
@@ -0,0 +1,193 @@
+# Send UDP Messages with OT CLI
+
+Thread networks offer several `udp` commands for establishing peer-to-peer communication
+using UDP sockets. Each peer needs its own socket.
+
+In addition to opening the required sockets, one requirement is that the socket
+on the receiving end of the initial communication be bound.
+
+The examples that follow demonstrate how you can open and bind sockets, how to
```

Comment:
Useful to mention that `cli udp` interact with one example socket.  

`cli udp` command does not really let user create new sockets .